content/vi/docs/reference/glossary/kube-proxy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: kube-proxy
+id: kube-proxy
+date: 2019-29-11
+full_link: /docs/reference/command-line-tools-reference/kube-proxy/
+short_description: >
+  `kube-proxy` là một tầng proxy mạng chạy trên mỗi node trong cluster.
+
+aka:
+tags:
+  - fundamental
+  - networking
+---
+
+[kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) là một tầng proxy mạng chạy trên mỗi node trong cluster, triển khai khái niệm về Kubernetes {{< glossary_tooltip term_id="service">}}.
```

Comment:
Must be: 
[kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) là một network proxy chạy trên mỗi node trong cluster, thực hiện một phần Kubernetes {{< glossary_tooltip term_id="service">}}.